scripts/lint_go_mod_version.go

```diff
@@ -0,0 +1,100 @@
+/*
+SPDX-FileCopyrightText: 2023 The Pion community <https://pion.ly>
+SPDX-License-Identifier: MIT
+*/
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+	"regexp"
+)
+
+func main() {
+	// Get the path of the current script
+	scriptPath, err := filepath.Abs(os.Args[0])
+	if err != nil {
+		log.Fatal(err)
+	}
+	scriptDir := filepath.Dir(scriptPath)
+
+	// Read the .ci.conf file if it exists
+	ciConfPath := filepath.Join(scriptDir, ".ci.conf")
+	if _, err := os.Stat(ciConfPath); err == nil {
+		if err := readCiConf(ciConfPath); err != nil {
+			log.Fatal(err)
+		}
+	}
+
+	// Read the go.mod file and extract the Go version information
+	goModPath := filepath.Join(scriptDir, "go.mod")
+	goModFile, err := os.Open(goModPath)
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer goModFile.Close()
+
+	scanner := bufio.NewScanner(goModFile)
+	re := regexp.MustCompile("^go\\s+([\\d.]+)$")
+	var goModVersion string
+	for scanner.Scan() {
+		line := scanner.Text()
+		if matches := re.FindStringSubmatch(line); matches != nil {
+			goModVersion = matches[1]
+			break
+		}
+	}
+	if err := scanner.Err(); err != nil {
+		log.Fatal(err)
+	}
+
+	expectedVersion := "1.19"
+	if goModVersion != expectedVersion {
+		if os.Getenv("CI") != "" {
+			goModVersionLine := getGoModVersionLine(goModPath, goModVersion)
+			fmt.Printf("::error title=Invalid Go version,file=%s,line=%d,::Found %s. Expected %s\n", goModPath, goModVersionLine, goModVersion, expectedVersion)
+		} else {
+			fmt.Printf("Invalid Go version in %s:\n", goModPath)
+			fmt.Printf("  Found    %s\n", goModVersion)
+			fmt.Printf("  Expected %s\n", expectedVersion)
+		}
+		os.Exit(1)
+	}
+}
+
+func readCiConf(ciConfPath string) error {
```

Comment:
```suggestion
func readCIConf(ciConfPath string) error {
```

In Go, acronyms are usually capitalized.